App/app/view/pivot/RoundRobinGrid.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * File: app/view/pivot/RoundRobinGrid.js
+ *
+ * This file was generated by Sencha Architect
+ * http://www.sencha.com/products/architect/
+ *
+ * This file requires use of the Ext JS 6.5.x Modern library, under independent license.
+ * License of Sencha Architect does not include license for Ext JS 6.5.x Modern. For more
+ * details see http://www.sencha.com/license or contact license@sencha.com.
+ *
+ * This file will be auto-generated each and everytime you save your project.
+ *
+ * Do NOT hand edit this file.
+ */
+
+Ext.define('Enif.view.pivot.RoundRobinGrid', {
+    extend: 'Ext.pivot.Grid',
+    alias: 'widget.pivot.roundrobingrid',
+
+    requires: [
+        'Enif.view.pivot.RoundRobinGridViewModel',
+        'Ext.pivot.matrix.Local',
+        'Ext.pivot.dimension.Item'
+    ],
+
+    viewModel: {
+        type: 'pivot.roundrobingrid'
+    },
+    height: '100%',
+    width: '100%',
+
+    matrix: {
+        store: 'PlayerData',
+        leftAxis: [
+            {
+                dataIndex: 'name',
+                sortIndex: 'uid'
+            }
+        ],
+        topAxis: [
+            {
+                dataIndex: 'name',
+                sortable: false,
+                sortIndex: 'uid'
+            }
+        ],
+        aggregate: {
+            renderer: function(value, record, dataIndex, cell, column) {
+                let cellColumn = parseInt(dataIndex.replace(/\D/g,''));
+                let playersStore = Ext.getStore('PlayerData');
+
+                if (cellColumn > playersStore.count() || record.data.isRowGrandTotal) { //aggregate cells
+                    cell.setBodyStyle({
+                        background: '#025b80'
+                    });
+                    return;
+                }
+
+                let cellRow = playersStore.find('name', record.data["ext-153"])+1;
```

Comment:
This looks weird. I don't think this is the best approach to the pivot grid. But can try to use that.